.prow.yaml

```diff
@@ -281,6 +281,22 @@ presubmits:
       imagePullSecrets:
       - name: quay
 
+  - name: pre-kubermatic-github-release
```

Comment:
Can we name this job, so it's clear it's dry-run? Maybe something along `pre-kubermatic-mock-github-release`.

Reply:
We can. I was thinking the `pre-` is enough, but apparently not. ^^

Reply:
It's `pre-kubermatic-simulate-github-release` now.